packages/hash/api/src/model/entityType.model.ts

```diff
@@ -103,17 +142,30 @@ class __EntityType {
     return new EntityType(entityType);
   }
 
-  static async updateSchema(
+  static async update(
```

Comment:
Might be out of scope of this PR, but I don't think this should be a static method. If you agree it'd be nice to change that or address it as part of a follow-up task.

Reply:
I agree with this, I have a question regarding the resolver:
Since we would have to fetch the `EntityType` if we make this change, should we then only allow updates if the user's `accountId` is equal to that of the `EntityType`? 
If I am not mistaken, it is currently possible to change `EntityType`s that the account did not create. What are your thoughts on this?

Reply:
My instinct is this is not to implement such a check now, as it'll be addressed when we implement the permissions model. That's when we should discuss where the best places would be to perform permission checks.

Right now you're right, a user can in theory update entities/links/entityTypes that aren't in an account they have access to. There's a basic check in the GQL queries that tries to prevent the user from doing things in other accounts, but there's likely edge cases this middleware isn't accounting for.